go.mod

```diff
@@ -39,4 +39,7 @@ require (
 	sigs.k8s.io/kind v0.11.1
 )
 
-replace github.com/sirupsen/logrus => github.com/projectcalico/logrus v1.0.4-calico
+replace (
+	github.com/sirupsen/logrus => github.com/projectcalico/logrus v1.0.4-calico
+	google.golang.org/grpc => google.golang.org/grpc v1.29.1
```

Comment:
Why do we need to replace the grpc library version?

Reply:
The latest version of `google.golang.org/grpc` that has `google.golang.org/grpc/naming` is `v1.29.1`.
The old version of etcd like v3.3.25 uses 'naming'. 
```
go: finding module for package google.golang.org/grpc/naming
github.com/projectcalico/libcalico-go/lib/backend/etcd imports
        go.etcd.io/etcd/client tested by
        go.etcd.io/etcd/client.test imports
        github.com/coreos/etcd/integration imports
        github.com/coreos/etcd/proxy/grpcproxy imports
        google.golang.org/grpc/naming: module google.golang.org/grpc@latest found (v1.40.0), but does not contain package google.golang.org/grpc/naming
```